src/DAL/repositories/recordRepository.ts

```diff
@@ -22,14 +22,16 @@ export class RecordRepository extends Repository<RecordEntity> {
 
   public async createRecord(req: IRasterCatalogUpsertRequestBody): Promise<string> {
     const entity = this.recordConvertor.createModelToEntity(req);
+    entity.id = req.id;
```

Comment:
the converter don't includes the id already?

Reply:
fixed